database/useCase1/application.sql

```diff
@@ -1,13 +1,13 @@
 -- SCHEMA: application
```

Comment:
The SQL files for the useCase1 seem to be using the tables from the initial dummy use case instead of the new use case 1 tables. Is that intended?

Reply:
Fixed with latest commits. Resolving!